src/test/java/org/junitpioneer/testkit/PioneerTestKit.java

```diff
@@ -46,4 +46,30 @@ public static ExecutionResults executeTestMethodWithParameterTypes(Class<?> test
 		return new ExecutionResults(testClass, testMethodName, methodParameterTypes);
 	}
 
+	/**
+	 * Returns the execution results of the given method of a given test class.
+	 *
+	 * @param testClass Name of the test class
+	 * @param testMethodName Name of the test method (of the given class)
+	 * @param methodParameterTypes Array containing the types of for the parameters
+	 * @return The execution results
+	 */
+	public static ExecutionResults executeTestMethodWithParameterTypes(Class<?> testClass, String testMethodName,
```

Comment:
These two methods should be merged by using the `Class<?>...` as the last parameter type. Which should throw an exception if no last parameter is passed (or maybe just call `executeTestMethod` as an attempt to "fix" the bad call?).

Alternatively, you could make it like this:
```
public static ExecutionResults executeTestMethodWithParameterTypes(Class<?> testClass, String testMethodName, Class<?> firstParameter, Class<?>... additionalParameters)
```
This ensures at least one class object is always passed to the method but is much more cumbersome. Method signature might be confusing too.